homeassistant/components/generic/config_flow.py

```diff
@@ -391,3 +426,39 @@ async def async_step_init(
             ),
             errors=errors,
         )
+
+
+class CameraImagePreView(CameraImageView):
+    """Camera view to temporarily serve an image."""
+
+    name = "api:camera:imgepreview"
```

Comment:
This should match the URL being used but with `:`

Reply:
Ok, done 39562b8c